bin/requirements-checker.php

```diff
@@ -17,15 +17,29 @@
 } elseif (file_exists($autoloader = __DIR__.'/../vendor/autoload.php')) {
     require_once $autoloader;
 } else {
-    throw new \RuntimeException('Unable to find the Composer autoloader.');
+    require_once dirname(__DIR__).'/src/Requirement.php';
+    require_once dirname(__DIR__).'/src/RequirementCollection.php';
+    require_once dirname(__DIR__).'/src/PhpConfigRequirement.php';
+    require_once dirname(__DIR__).'/src/SymfonyRequirements.php';
 }
 
 $lineSize = 70;
-$isVerbose = in_array('-v', $argv) || in_array('-vv', $argv) || in_array('-vvv', $argv);
+$args = array();
+$isVerbose = false;
+foreach ($argv as $arg) {
+    if ('-v' === $arg || '-vv' === $arg || '-vvv' === $arg) {
+        $isVerbose = true;
+    } else {
+        $args[] = $arg;
+    }
+}
 
 $symfonyVersion = class_exists('\Symfony\Component\HttpKernel\Kernel') ? \Symfony\Component\HttpKernel\Kernel::VERSION : null;
 
-$symfonyRequirements = new SymfonyRequirements(dirname(dirname(realpath($autoloader))), $symfonyVersion);
+// specific directory to check?
+$dir = isset($args[1]) ? $args[1] : dirname(dirname(realpath($autoloader)));
```

Comment:
This allows to have only one checkout of the repository and run it on different directories.